libr/include/r_crypto/r_bech32.h

```diff
@@ -0,0 +1,19 @@
+#ifndef R_BECH32_H // is this file still useful ?
```

Comment:
i guess we can remove this file completely, because it doesnt provide any define or enum